stable/envoy/README.md

```diff
@@ -26,16 +26,24 @@ The command removes all the Kubernetes components associated with the chart and
 
 The following table lists the configurable parameters of the envoy chart and their default values.
 
-Parameter | Description | Default
---- | --- | ---
-`files.envoy\.yaml` | content of a full envoy configuration file as documented in https://www.envoyproxy.io/docs/envoy/latest/configuration/configuration | See [values.yaml](values.yaml)
-`templates.envoy\.yaml` | golang template of a full configuration file. Use the `{{ .Values.foo.bar }}` syntax to embed chart values | See [values.yaml](values.yaml)
-`serviceMonitor.enabled` | if `true`, creates a Prometheus Operator ServiceMonitor | `false`
-`serviceMonitor.interval` | Interval that Prometheus scrapes Envoy metrics | `15s`
-`serviceMonitor.namespace` | Namespace which the operated Prometheus is running in | ``
-`serviceMonitor.additionalLabels` | Labels used by Prometheus Operator to discover your Service Monitor. Set according to your Prometheus setup | `{}`
-| `prometheusRule.enabled` | If `true`, creates a Prometheus Operator PrometheusRule | `false``
-| `prometheusRule.groups` | Prometheus alerting rules | `{}`
-| `prometheusRule.additionalLabels` | Labels used by Prometheus Operator to discover your Prometheus Rule | `{}`
+| Parameter                         | Description                                                                                                                                                       | Default                                           |
```

Comment:
Only other suggestion is keep the old table format and only add the new values. The nice thing about just using the minimum (value, space, pipe, value) is when charts change the entire readme doesn’t need to reformat to align the longest string in a column. It keeps changesets smaller and more pertinent to each new change. If you don’t mind changing that table format back the rest of this looks great

Reply:
That makes sense! Addressed in 3355cc6